### PR TITLE
Models from Proposal

### PR DESCRIPTION
**Pull request by ceguiluzrosas**
### Copying Down from Proposal
**Note**: 
- To run, do: 
```bash
cd backend
# make sure that the db.sqlite3 file is not present in repo or hasn't had any migrations applied yet
# otherwise, delete both db.sqlite3 and migrations/0001_initial.py file
python3 manage.py makemigrations api
python3 manage.py migrate 
```
- Check off any models that are ready to go 👍🏽 
- Will address the OAuth2.0 aspect in another PR; following this [guide](https://django-oauth-toolkit.readthedocs.io/en/latest/index.html)
----
*In our actual database, all of our different “users” will be in one user table with permissions, roles, etc.
- [x] User
  - [x] Name
  - [x] Email
  - [ ] *We will not be needing to store the password because we will be using OAuth 2.0
  - [x] List of stores they are associated with (to be linked with stores by a unique store id)
- [x] Store:
  - [x] Unique Store ID
  - [x] Store Information
  - [x] Address
  - [x] Name
  - [x] Category
- [x] Items:
  - [x] Image
  - [x] Stock of each item
  - [x] (Optional) Price of each item
  - [x] Description of each item
  - [x] List of users associated with the store
  - [x] Role of each user
  - [x] User membership (start_date)
  - [x] Permission of each type of user (manager, vendor, employee)

### CODE REVIEW

**Comment by MattKersey** [comment]
When accessing the /users path, I get an error:
Screen Shot 2020-11-11 at 6 15 57 AM
After doing some digging, I think this is probably tied to line 7 in serializers.py. Changing <code>model = get_user_model</code> to <code>model = get_user_model()</code> fixed the issue for me. Everything else looks great as far as I can tell.

**Review by MattKersey — CHANGES_REQUESTED** [review]
See comment from conversations.

File: backend/test_dummy.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 def test_dummy():
-    assert True == True
+    assert 1+1 == 2
```
Comment (ceguiluzrosas) [inline backend/test_dummy.py:2]: Flake8 was giving me an error for this

**Comment by ceguiluzrosas** [comment]
@MattKersey @jkim10 @yiqiao-liu So I was working to address the change the Matt had mentioned and along the way I realized that something was wrong with the setup of the rest-framework package. I didn't want to hold the project any longer, so I simplified the project to not use the package **for now** so that we can move the team forward in backend-development. I'll try to address the rest_framework issue later. Hope y'all can understand.

**Comment by yiqiao-liu** [comment]
> @MattKersey @jkim10 @yiqiao-liu So I was working to address the change Matt had mentioned and along the way I realized that something was wrong with the setup of the rest-framework package. I didn't want to hold the project any longer, so I simplified the project to not use the package **for now** so that we can move the team forward in backend-development. I'll try to address the rest_framework issue later. Hope y'all can understand.

Sounds quite reasonable to me!

**Comment by ceguiluzrosas** [comment]
@MattKersey @yiqiao-liu @jkim10 Should we merge?

**Review by MattKersey — APPROVED** [review]
Looks good to me, seems to be working.

**Pull request merged**